### PR TITLE
#161838472 Validation Script bugfix

### PR DESCRIPTION
**Pull request by ajimae**
#### What does this PR do?
- Fixes bug on user sign up validation
- Add a maximum length rule of 100 to `.eslint`
#### Description of Task to be completed?
- Checkout out a new branch from develop branch
- Fix bug on line 96 column 16 on validation file
- Raise a PR of the branch again develop
- Deploy new branch to Heroku and obtain the review app url.
- Test url on Postman to ensure issues was fixed.

#### How should this be manually tested?
- Clone the `bug/161838472/valiation-script-bugfix` to local machine
- Run `npm install`
- Go to `localhost:3000/api/v1/users/register` and enter only `username` and `confirmPassword` into the body.

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- 161838472

#### Screenshots (if appropriate)

image

### CODE REVIEW

**Review by ovieokeh — APPROVED** [review]
A very timely fix to a serious problem. Merge it already!!

**Review by desaawa — APPROVED** [review]
When you resolve merge conflicts, go ahead and merge.

File: server/src/controllers/SearchController.js [hunk 1]
```diff
@@ -41,12 +44,15 @@ class SearchController {
...
-    const query = `${q1}${q2}${q3}${q4}${q5}`;
+    const q1 = 'SELECT "Articles".id, "Articles"."userId", "Articles".title,';
+    const q2 = ' "Articles".description,  "Articles".content, "Articles".slug,';
+    const q3 = ' "Articles"."featuredImg", "Articles"."createdAt", ';
+    const q4 = '"Articles"."createdAt", "Articles"."updatedAt" FROM "Articles"';
+    const q5 = ' INNER JOIN "Users" ON "Articles"."userId" = "Users"."id" AND ';
+    const q6 = '"Articles".published = \'TRUE\' WHERE "Users"."username" =';
+    const q7 = ` '${filters.author}' AND "Articles"."searchVectors" @@ to_tsquery('${searchTerm}')`;
+    const q8 = 'ORDER BY "Articles"."createdAt" DESC';
+    const query = `${q1}${q2}${q3}${q4}${q5}${q6}${q7}${q8}`;
```
Comment (andela-moe) [inline server/src/controllers/SearchController.js]: @ajimae @65black  I believe we could have found a better way of doing this. Not sure right now.  I will go ahead to merge this now but please ensure that you discuss this with your TTL.

**Comment by andela-moe** [comment]
@ajimae Conflicts on this PR. Please resolve.

**Pull request merged**